api/src/main/resources/liquibase.xml

```diff
@@ -671,4 +671,29 @@
             ALTER TABLE patient_appointment_provider MODIFY voided TINYINT(1) DEFAULT 0;
         </sql>
     </changeSet>
+    <changeSet id="create-column-priority-for-appointment-202304191703" author="Kavitha, Umair">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="patient_appointment" />
+            <not>
+                <columnExists tableName="patient_appointment" columnName="priority" />
+            </not>
+        </preConditions>
+        <addColumn tableName="patient_appointment">
+            <column name="priority" type="varchar(45)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="remove-not-null-constraint-202304211826" author="Kavitha, Umair">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="patient_appointment" />
+            <columnExists tableName="patient_appointment" columnName="start_date_time" />
+            <columnExists tableName="patient_appointment" columnName="end_date_time" />
+        </preConditions>
+        <comment>Update start and end date time of patient appointment table to be nullable</comment>
+        <sql>
+            ALTER TABLE patient_appointment MODIFY start_date_time datetime;
```

Comment:
I think its better to be explicit about default as null using `DEFAULT NULL`.

https://fedingo.com/how-to-modify-mysql-column-to-allow-null/

Reply:
default value is set to NULL now